pkg/apis/pingcap/v1alpha1/tidbcluster.go

```diff
@@ -27,7 +27,7 @@ import (
 
 const (
 	// defaultHelperImage is default image of helper
-	defaultHelperImage        = "busybox:1.26.2"
+	defaultHelperImage        = "busybox:1.34.1"
```

Comment:
We cannot update the default image here, which will cause the rolling update of TiDB after upgrading the TiDB Operator, and is not a good practice.
This is configurable in the CR so if the user wants to use arm64 image, they can just configure it.

Reply:
I understand that we want to be careful with existing installations. However for new installations (especially on ARM64, but also on any other platform) having and old and possibly insecure busybox versions isn't great.

Can we modify this to use 1.34.1 for new installations while not updating older installations?

Would it be ok to set it to 1.34.1 in all example files (including those where currently no version is specified)?

Is this something that can be done in a new major version of the operator?

Reply:
@dveeden I understand your concern.
Even if with the major version, we still prefer not to trigger a rolling update, as you know that TiDB connections are not handled well during rolling update.

I think it's OK `to set it to 1.34.1 in all example files (including those where currently no version is specified)`, could you please help to improve the examples?

Reply:
@DanielZhangQD done

Not sure what's needed for `charts/tidb-cluster/values.yaml` etc.

Reply:
I've updated `charts/tidb-cluster/templates/tidb-cluster.yaml` and `charts/tidb-cluster/values.yaml`.

Reply:
The `tidb-cluster` chart has been deprecated since v1.1.0 but your change is OK, thanks!